recipes/python-pdal/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "python-pdal" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "60d41da79441d0aa9c44ec6ebb7497ffc0090d761259d42d4eeb05c7146bb432" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/PDAL/python/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py!=36]
```

Comment:
The README in the repo seems to indicate this will work with python 2.7 and any version of 3. Any reason to skip those builds?

Reply:
This is a consequence of the PDAL dependency also depending on Python. There wasn't a way IIRC to sync up the Python versions across packages. If I'm wrong, just let me know. It should be noted that it is on our roadmap to move all Python dependencies in the current PDAL codebase to the PDAL/python codebase (it will build PDAL plugins plus the Python bindings), so this problem goes away at that time, and we can unlock the python 2.7 builds.

Reply:
Ok, I can see that `pdal` was only built for 3.6.

Reply:
Wait, so `pdal` actually is built for a specific Python version? Having `python` only in `build` and not in `run`, as the `pdal` recipe does, implies that the package should be independent of the Python version.

Reply:
Currently, yes, though it will be moving (see https://github.com/PDAL/PDAL/issues/1981). Are you suggesting that we have `python` only in `build` here to match the PDAL recipe?

Reply:
No, I'm suggesting that the metadata of the PDAL recipe is broken if it actually depends on the python version. :)